### DISCUSSION TITLE
Competitors

### DISCUSSION THREAD

**Discussion by dnecklesportfolio** [SelfControlApp/selfcontrol#865] [General]
Does another app like Self Control exist? For apps that block overall websites I am open to suggestions if any exist.

Thank you

**Comment by slambert** [comment]
There are a few. We mention some in the [FAQ](https://github.com/SelfControlApp/selfcontrol/wiki/FAQ#q-are-you-planning-on-making-a-windows-linux-or-ios-version)

This is pretty popular, commercial software: [Cold Turkey](https://getcoldturkey.com/features/)

There's also [Freedom](https://freedom.to/)

And you could [skim articles like this one.](https://zapier.com/blog/stay-focused-avoid-distractions/)
